WBooks/BookDetails/InfoBookSection/InfoBookSectionController.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  InfoBookSectionController.swift
+//  WBooks
+//
+//  Created by Juan Silva on 14/06/2021.
+//
+
+import Foundation
+import UIKit
+
+class InfoBookSectionController: UIViewController {
+    private lazy var infoBookSectionView = InfoBookSectionView()
+    private let infoBookSectionViewModel: InfoBookSectionViewModel
+        
+    init(bookSectionViewModel: InfoBookSectionViewModel) {
+        self.infoBookSectionViewModel = bookSectionViewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func loadView() {
+        super.loadView()
+        self.view = infoBookSectionView
+        infoBookSectionView.addToWishListButton.setTitle(NSLocalizedString("ADD_TO_WISHLIST_BUTTON", comment: "Add to wishlist text button"), for: .normal)
+        infoBookSectionView.rentButton.setTitle(NSLocalizedString("RENT_BUTTON", comment: "Rent text button"), for: .normal)
```

Comment:
Usually the configuration of the view is done after the view loads, so let's move these lines to `viewDidLoad()`.